backend/backends/null/null.go

```diff
@@ -0,0 +1,39 @@
+package null
+
+import (
+	"github.com/atlassian/gostatsd/backend"
+	"github.com/atlassian/gostatsd/types"
+
+	"github.com/spf13/viper"
+)
+
+const backendName = "null"
+
+func init() {
```

Comment:
Could you elaborate more please? I don't understand :) I just copied some existing backend and removed everything except method signatures.